src/parseutils.jl

```diff
@@ -22,27 +22,4 @@ group(i, re::Regex, bytes, default) =
     default :
     SubString(bytes, re.ovec[2i+1]+1, prevind(bytes, re.ovec[2i+2]+1))
 
-function lceq(a, b)
-    astate = iterate(a)
-    bstate = iterate(b)
-    if astate === nothing
-        return bstate === nothing
-    elseif bstate === nothing
-        return false
-    end
-    ac::Char, ast = astate
-    bc::Char, bst = bstate
-    while true
-        lowercase(ac) === lowercase(bc) || return false
-        astate = iterate(a, ast)
-        bstate = iterate(b, bst)
-        if astate === nothing
-            return bstate === nothing
-        elseif bstate === nothing
-            return false
-        end
-        ac, ast = astate
-        bc, bst = bstate
-    end
-    return true
-end
\ No newline at end of file
+lceq(a, b) = lowercase(a) == lowercase(b)
```

Comment:
Needs newline at end of file.

Should put the FIXME notes back in:
```julia
# FIXME consider allocation and speed efficiency of lowercase comparisons		
 # - Make a LCString <: AbstractString wrapper that translates in-line?		
 # - Make a lcmp function that translates as it goes?		
 # https://github.com/JuliaLang/julia/issues/19972
```

Also, should extend the FIXME to say:
```julia
# Should take advantage of the fact that only ascii characters are allowed
# in HTTP field-name.
# i.e. it is probably better to iterate over codeunits and use something like:
# asciilc(c) = c in UInt8('A'):UInt8('Z') ? c + 0x20 : c
```

Reply:
Those fixme nots should just be Issue in the github repository.
Comments in the code  are not a good long term place for storing that kinda info.

Reply:
I'm happy for these to be moved into an issue and replace the FIXME with: `#FIXME See https://github.com/JuliaWeb/HTTP.jl/issues/XXX`

Reply:
https://github.com/JuliaWeb/HTTP.jl/issues/310. I don't think adding a comment in the code is helpful at all and is much more likely to be lost and forgotten than a github issue.